src/cswinrt/code_writers.h

```diff
@@ -5896,14 +5913,36 @@ public static Guid PIID = Vftbl.PIID;
 
     void write_static_abi_classes(writer& w, TypeDef const& iface)
     {
+        auto fast_abi_class_val = get_fast_abi_class_for_interface(iface);
+        if (fast_abi_class_val.has_value())
+        {
+            if (!interfaces_equal(fast_abi_class_val.value().default_interface, iface))
+            {
+                return;
+            }
+        }
+
         w.write(R"(% static class %
 {
 %
 }
 )", 
         is_exclusive_to(iface) ? "internal" : internal_if_embedded(), 
         bind<write_type_name>(iface, typedef_name_type::StaticAbiClass, false), 
-        bind<write_static_abi_class_members>(iface));
+        [&](writer& w) {
+            if (!fast_abi_class_val.has_value()) {
+                write_static_abi_class_members(w, iface, INSPECTABLE_METHOD_COUNT);
+                return;
+            }
+            auto abi_methods_start_index = INSPECTABLE_METHOD_COUNT;
+            write_static_abi_class_members(w, fast_abi_class_val.value().default_interface, abi_methods_start_index);
```

Comment:
Is this supposed to be the default interface?  And how are the static and factory interface methods ordering offset with the other exclusive interface methods?  And do we have any fast abi tests for static methods and factory methods?